scripts/kind-e2e/e2e.sh

```diff
@@ -322,16 +331,108 @@ setup_custom_cni
 if [[ $3 = true ]]; then
     enable_logging
 fi
-install_helm
-if [[ $4 = true ]]; then
```

Comment:
So no kubefed when using the operator?

Reply:
This is fixed in the latest version, thanks.